### PR TITLE
Add setup and license

### LINKED ISSUE #1: Licence
- [ ] Decide on a licence
- [ ] Create licence file
- [ ] Add licence line(s) to [README.md](https://github.com/SciTools-incubator/iris-ugrid/blob/master/README.md)

### PR DESCRIPTION
**Pull request by alastair-gemmell**
closes #1 and adds setup.py and setup.cfg

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+
+
+from setuptools import setup
```
Comment (trexfeathers) [inline setup.py:4]: setuptools should be added to the [dependencies](https://github.com/SciTools-incubator/iris-ugrid/blob/62c8e27b1e3f46b4042605225ae5e00be27b630f/requirements/conda_requirements.txt)

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+zip_safe = False
+
+[tool:pytest]
+testpaths =
+    iris_ugrid/
+addopts =
+    -ra
+    -v
+    --doctest-modules
+doctest_optionflags = NORMALIZE_WHITESPACE ELLIPSIS NUMBER
```
Comment (trexfeathers) [inline setup.cfg:45]: I think most software likes a new line at the end. This has been picked up in the GitHub diff anyway

**Review by trexfeathers — CHANGES_REQUESTED** [review]
I am happy that these files closely mirror those in [SciTools-incubator/iris-esmf-regrid](https://github.com/SciTools-incubator/iris-esmf-regrid), which is desirable.

I have also confirmed that [airspeed velocity](https://asv.readthedocs.io/en/stable/index.html) is now able to work with the repo in this form, which is highly encouraging since [it makes varied use of setup files by default](https://asv.readthedocs.io/en/stable/asv.conf.json.html#build-command-install-command-uninstall-command).

Couple of minor changes if you don't mind @alastair-gemmell.

@pp-mo did you want me to hold off merging until you have also reviewed?

File: setup.py (hunk 1 above)
Comment (trexfeathers) [reply setup.py:4]: (I will shortly be aligning the dependency structure with the structure already used in Iris and elsewhere - see #31 - so no need to be concerned with the simple structure at this point)

Comment (alastair-gemmell) [reply setup.py:4]: Well spotted - thanks @trexfeathers. Done this now.

File: setup.cfg (hunk 2 above)
Comment (alastair-gemmell) [reply setup.cfg]: Another good spot thanks! Have added a newline now

**Review by trexfeathers — APPROVED** [review]

**Pull request merged**